CODINGSTYLE.md

```diff
@@ -14,13 +14,57 @@ In these cases, it can be overridden by surrounding it with `clang-format-off` a
 
 ### Naming convention
 
-* Almost everything uses `lower_snake_case`.
+* Local variables uses `lower_snake_case`.
 * `#define` uses `UPPER_SNAKE_CASE`.
 * Everything is in a namespace.
 * Therefore, there are no prefixes.
-* Private data members have an "m" prefix: `m_data`.
-* Getter functions use the name without "m": `data()`.
-* Setter functions use the "set" prefix: `set_data(value)`.
+
+### Objects types
```

Comment:
The text below is really still about naming convention. Even "when to use struct and when to use class" can be considered naming convention. So I wouldn't add this subtitle and just make the rest subsections of Naming convention.

Reply:
Ok, Removed the title.